lib/protobuf/protoc/generator.ex

```diff
@@ -27,13 +27,13 @@ defmodule Protobuf.Protoc.Generator do
 
       content =
         module_definitions
-        |> Enum.map(fn {_mod_name, contents} -> [contents, ?\n] end)
-        |> IO.iodata_to_binary()
+        |> Enum.map(fn {_mod_name, contents} -> contents end)
+        |> Enum.join("\n\n")
```

Comment:
The alternative way would be using `Protobuf.Protoc.Generator.Util.format/1` instead.